frontend/src/features/admin-form/settings/components/FormNricMaskingToggle.tsx

```diff
@@ -0,0 +1,47 @@
+import { useCallback, useMemo } from 'react'
+import { Icon, Skeleton } from '@chakra-ui/react'
+
+import { BxsInfoCircle } from '~assets/icons'
+import Toggle from '~components/Toggle'
+import Tooltip from '~components/Tooltip'
+
+import { useMutateFormSettings } from '../mutations'
+import { useAdminFormSettings } from '../queries'
+
+export const FormNricMaskingToggle = (): JSX.Element => {
+  const { data: settings, isLoading: isLoadingSettings } =
+    useAdminFormSettings()
+
+  const isNricMaskingEnabled = useMemo(
+    () => settings && settings.isNricMaskingEnabled,
```

Comment:
/nit can be terser with `settings?.isNricMaskingEnabled`

Reply:
Yeah, that makes sense. 
I see a settings && settings?.authType check in other toggle.tsx files, perhaps we can do a refactor in the future in another commit to replace with just settings?.authType